x-pack/plugins/lists/public/exceptions/components/builder/helpers.ts

```diff
@@ -591,7 +591,7 @@ export const getEntryOnOperatorChange = (
           },
           ...parent.parent.entries.slice(entryIndex + 1),
         ],
-      },
+      } as BuilderEntry,
```

Comment:
I think you can delete this. I tried deleting it and didn't get an error.

Reply:
Quite right. This was needed when entriesArray had 'nested' in it.

Reply:
done 0bf81993a3645cb4a493fa82e717bec675ba0ecc